packages/venia-concept/jest.config.js

```diff
@@ -21,8 +20,10 @@ module.exports = {
         // improves
         '^@magento/peregrine(/*(?:.+)*)': join(__dirname, '../peregrine/src/$1')
     },
+    testPathIgnorePatterns: ['dist', 'node_modules'],
+    testURL: 'https://localhost/',
     // Have Jest use Babel to transpile Peregrine imports in tests, since
     // our cross-package tests in the monorepo should all operate on `src`
     transformIgnorePatterns: ['node_modules/(?!@magento/peregrine)'],
-    testPathIgnorePatterns: ['dist', 'node_modules']
+    setupTestFrameworkScriptFile: '<rootDir>/jest.setup.js'
```

Comment:
More alphabetical changes here. The only real addition is the `setupTstFrameworkScriptFile` property.